jetson/nav/search/diamondSearch.txt

```diff
@@ -0,0 +1,9 @@
+0.0 90.0
```

Comment:
Can we delete this? It seems redundant to have, plus we're removing the "DiamondSearch" we have in this PR